src/control/cmd/drpc_test/hello/drpc_test.pb.go

```diff
@@ -7,7 +7,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.26.0
-// 	protoc        v3.6.1
+// 	protoc        v3.12.4
```

Comment:
We generally refrain from committing these kind of version-only changes to the generated protobuf files, as they contribute to churn and increase the chance of conflicts. I think in this case, though, I'm fine with letting these changes stay in as I think most developers have moved on to this version of the toolchain.